DirectProgramming/C++SYCL_FPGA/Tutorials/Features/ip_authoring_interfaces/component_interfaces_comparison/mm-host/src/vector_add.cpp

```diff
@@ -0,0 +1,141 @@
+#include <iostream>
+
+// oneAPI headers
+#include <sycl/sycl.hpp>
+#include <sycl/ext/intel/fpga_extensions.hpp>
+#include "exception_handler.hpp"
+
+// Buffer locations for MM Host interfaces
+constexpr int kBL1 = 1;
+constexpr int kBL2 = 2;
+constexpr int kBL3 = 3;
+
+// Forward declare the kernel name in the global scope. This is an FPGA best
+// practice that reduces name mangling in the optimization reports.
+class SimpleVAdd;
+
+struct SimpleVAddKernel {
+  sycl::ext::oneapi::experimental::annotated_arg<
+      int *, decltype(sycl::ext::oneapi::experimental::properties{
+                 sycl::ext::intel::experimental::buffer_location<kBL1>,
+                 sycl::ext::intel::experimental::dwidth<32>,
+                 sycl::ext::intel::experimental::latency<0>,
+                 sycl::ext::intel::experimental::read_write_mode_read,
+                 sycl::ext::intel::experimental::wait_request_requested})>
```

Comment:
Remove waitreq from all your samples.
```suggestion
                 })>
```